### ISSUE TITLE
Memory leak

### ISSUE THREAD

**Issue by samreid** [phetsims/circuit-construction-kit-common#775]
In https://github.com/phetsims/qa/issues/736#issuecomment-964526145 @KatieWoe said:

> I did a quick memory test to see if there were any leaks. @samreid do these results look ok?
<img width="344" alt="memtest" src="https://user-images.githubusercontent.com/41024075/141001192-892fc4be-1001-4d43-9811-3ac14e4080c2.png">

**Comment by samreid** [comment]
@KatieWoe was that in PhET brand or PhET-iO brand?

**Comment by KatieWoe** [comment]
This was PhET brand

**Comment by samreid** [comment]
For this playgound sim, I used the following strategy to memory leak test.

1. Start sim with ?fuzz and watch it fuzz about 10 seconds.  For a warm-up period.
2. Run `phet.chipper.queryParameters.fuzz=!phet.chipper.queryParameters.fuzz` to disable fuzzing
3. Navigate to each screen and press the reset all button.
4. Take a memory snapshot.
5. Run `phet.chipper.queryParameters.fuzz=!phet.chipper.queryParameters.fuzz` and `setTimeout(()=>{phet.chipper.queryParameters.fuzz=!phet.chipper.queryParameters.fuzz},60*1000);` to fuzz for one minute.
6. Reset each screen and take another memory snapshot.

This helped me track down memory leaks due to the series ammeter node and the dog node.  There are other suspicious parts in common code, but at such a low rate that it doesn't seem problematic.  

![image](https://user-images.githubusercontent.com/679486/141507237-b1cb20ed-f149-44b8-8f6a-0a06b0d52122.png)

Closing.
